core/test/audits/__snapshots__/metrics-test.js.snap

```diff
@@ -16,6 +16,8 @@ Object {
   "largestContentfulPaintAllFrames": 683,
   "largestContentfulPaintAllFramesTs": 23466705983,
   "largestContentfulPaintTs": 23466886143,
+  "lcpLoadEnd": 565,
```

Comment:
all these tests either have undefined lcpLoadStart/End or === to ttfb, so maybe helpful to add a new case to metrics-test?